frontend/src/routes/(user)/signup/+page.svelte

```diff
@@ -48,7 +49,14 @@
 			label="Repeat password"
 			class="w-56 mb-3"
 		/>
-		<Button type="submit" variant="raised" class="mt-5">Sign Up</Button>
+		<Button
+			type="submit"
+			variant="raised"
+			class="mt-5"
+			on:click={() => amplitude.track('User Registered')}
```

Comment:
Shouldn't we track this only if registration was successful?

Reply:
Hmm probably, but the amplitude events don't work well in the backend js. Maybe since we track login events amplitude can infer registrations and we can remove this.